### DISCUSSION TITLE
weed shell on S3 - just gets . (dot/point)

### DISCUSSION THREAD

**Discussion by shuther** [seaweedfs/seaweedfs#5507] [Q&A]
I had seaweedfs running fine with docker compose (based on original version with 2 volumes), but I am trying to troubleshoot an error.
I tried to connect to the s3 'shell' using:
```
docker compose exec s3 sh
weed shell
```
but in the reply, I am only getting dots (even after 2 hours)
...........................................................................................................................................................................................................................................................................................................................................................................................................................................................................

How could I debug the problem further?
On the master, I have:
```
>  fs.verify 
total 0 directories, 0 files
verified 0 files, error 0 files 
>  lock
>  volume.fsck 
total 0 directories, 0 files
dataNode:192.168.48.4:8080	volume:7	entries:849	orphan:849	100.00%	3090100752B
dataNode:192.168.48.4:8080	volume:3	entries:865	orphan:865	100.00%	3158649514B
dataNode:192.168.48.4:8080	volume:2	entries:843	orphan:843	100.00%	3108006373B
dataNode:192.168.48.4:8080	volume:5	entries:862	orphan:862	100.00%	3199993555B
dataNode:192.168.48.4:8080	volume:4	entries:905	orphan:905	100.00%	3213587111B
dataNode:192.168.48.4:8080	volume:1	entries:901	orphan:901	100.00%	3178898470B
dataNode:192.168.48.4:8080	volume:6	entries:875	orphan:875	100.00%	3213472790B
dataNode:192.168.48.5:8080	volume:12	entries:29	orphan:29	100.00%	77218443B
dataNode:192.168.48.5:8080	volume:14	entries:35	orphan:35	100.00%	60746921B
dataNode:192.168.48.5:8080	volume:9	entries:49	orphan:49	100.00%	97774022B
dataNode:192.168.48.5:8080	volume:10	entries:33	orphan:33	100.00%	8513887B
dataNode:192.168.48.5:8080	volume:13	entries:29	orphan:29	100.00%	25055111B
dataNode:192.168.48.5:8080	volume:8	entries:37	orphan:37	100.00%	166661942B
dataNode:192.168.48.5:8080	volume:11	entries:30	orphan:30	100.00%	81240049B

Total		entries:6342	orphan:6342	100.00%	22679918940B
This could be normal if multiple filers or no filers are used.
```

In the log, I face the error below (last line from docker compose):

> filer-1           | I0417 11:08:01.988360 filer_server_handlers_read_dir.go:102 Template Execute Error: write tcp [::1]:8888->[::1]:39920: write: connection reset by peer

**Comment by chrislusf** [comment]
You can use "weed shell -h" to see the options to specify the master. Default is "localhost:9333", which does not always work on other machines.
